CONTRIBUTING.md

```diff
@@ -0,0 +1,56 @@
+# Contributing to `nexus-sdk`
+
+Thank you for considering contributing to this project!
+The following guidelines help ensure a smooth process for everyone.
+
+## CLA
+
+Contributors are required to sign the [CLA]. This process is handled automatically by the [CLA Assistant].
+
+## Code of Conduct
+
+This project adheres to a [Code of Conduct]. By participating, you are expected to uphold this code.
+
+## How to Contribute
+
+1. **Fork the repository** and create your branch from `main`.
+2. **Follow the coding style** used in the project.
+3. **Write clear, concise commit messages.**
```

Comment:
I'd add that both the commit messages and PR titles should follow the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) guideline.